### PR TITLE
Issues 86 & 87 : import and export boards

### PR DESCRIPTION
**Pull request by vrialland**

### CODE REVIEW

File: kansha/board/boardsmanager.py [hunk 1]
```diff
@@ -7,52 +7,27 @@
...
 import os.path
 import json
-import time
-
-from nagare import database
 
 from .models import DataBoard
-from ..label.models import DataLabel
-from ..card.models import DataCard
 from ..card.fts_schema import Card as FTSCard
```
Comment (rtxm) [inline kansha/board/boardsmanager.py]: import order and avoid relative imports of upper levels.

File: kansha/card/models.py [hunk 2]
```diff
@@ -16,6 +19,8 @@
 import datetime
```
Comment (rtxm) [inline kansha/card/models.py]: Misplaced import

File: kansha/board/comp.py [hunk 3]
```diff
@@ -886,6 +898,15 @@ def reload_search(self):
         self.must_reload_search = False
         return self.search(self.last_search)
 
+    def save_as_template(self, shared):
+        data = self.data.to_template(shared)
+        fname = unicodedata.normalize('NFKD', self.data.title).encode('ascii', 'ignore')
+        fname = re.sub('\W+', '_', fname.lower())
+        fname = '%s.%s.btpl' % (security.get_user().username, fname)
```
Comment (rtxm) [inline kansha/board/comp.py]: You shoud protect the username too.

Comment (nagareproject) [inline kansha/board/comp.py]: Don't use usernames as uid : user names are unicode, can have special chars inside ('nng/apr'), are not unique and not persistent (can change during somebody's life)

Comment (rtxm) [inline kansha/board/comp.py]: What about using the board id as namespace then?

Comment (nagareproject) [inline kansha/board/comp.py]: Store all the relations (template UUID / user / board) in the DB. Use the `asset manager` to physically store the template on disk and to affect it its UUID

**Pull request closed without merging**